### PR TITLE
Update 1.2.0 io page to use s3 filesystem

### PR DESCRIPTION
**Pull request by pgerv12**

### CODE REVIEW

File: docs/beamrunner/release/1.2.0/io.md [hunk 1]
```diff
@@ -14,89 +14,75 @@ next:
...
-virtualenv my_project
-cd my_project
-source bin/activate
-pip install python-swiftclient
-pip install python-keystoneclient
-```
+| Environment variable | Credentials field | Environment variable example|
+|----------------------|-------------------|-----------------------------|
+| `AWS_ACCESS_KEY_ID` | `cos_hmac_keys.access_key_id` | `export AWS_ACCESS_KEY_ID=de4e2e3d7bd943a99b672f13dec40f7c`|
+| `AWS_SECRET_ACCESS_KEY` | `cos_hmac_keys.secret_access_key` | `export AWS_SECRET_ACCESS_KEY=54f077c504ebef49bf707cc0d57e3f2a4f4d4a6898b53fec`|
```
Comment (ghost) [inline docs/beamrunner/release/1.2.0/io.md]: Should we describe how to get these values to the app with the `--awsCredentialsProvider` option?

**Comment by pgerv12** [comment]
@johnmac-ibm Yes, I'm up in the air about how to go about doing this. Should we put all that information in this location or should we split off to a new s3/Object Storage page where we go over these instructions and how to launch example (essentially move this content to `Using IBM Object Storage` page and link to that from this page.

File: docs/beamrunner/release/1.2.0/io.md [hunk 2]
```diff
@@ -14,89 +14,75 @@ next:
...
-| `OS_PASSWORD`          | `--swiftPassword`  | `password`          | `export OS_PASSWORD='f1H/~BIO.=s0wuT9'`                  |
-| `OS_PROJECT_ID`        | `--swiftProjectId` | `projectId`         | `export OS_PROJECT_ID='80301e24254f4ffb81d53f0cddccad78'` |
-| `OS_REGION_NAME`       | `--swiftRegion`    | `region`            | `export OS_REGION='dallas'`                               |
+To use the storage service from Beam applications, you must specify the IBM Cloud service credentials. Since the `s3` FileSystem is based on AWS object storage, you will need to create credentials with HMAC keys.
 
-**Tip**: For MacOS, the Swift command of OpenStack might collide with the existing Xcode Swift command. To avoid the conflicts, create a Python virtual environment, and install the Swift client in the virtual environment.
+1. On the service page, click **Service credentials**.
+1. Create a credential by clicking **New credential**.
+1. Put `{"HMAC":true}` in the **Add Inline Configuration Parameters (Optional) field** and click **Add** to create the new credentials.
```
Comment (mrshenli) [inline docs/beamrunner/release/1.2.0/io.md]: Question: what is "HMAC"?

Comment (pgerv12) [reply docs/beamrunner/release/1.2.0/io.md]: Unfortunately the docs do not mention it, but I found "Hash Message Authentication Code" that seems to fit

**Comment by ghost** [comment]
I guess it depends how long those instructions are. If they are "short enough" (whatever that means) we can inline them, but if not I think we should at least mention how the credentials are used here with the reference to the full example. Something like:

> These credentials will be used with the `--awsCredentialsProvider` pipeline option, see _Running `FileStreamSample` with Object Storage_ for detailed instructions.

Maybe it should be in `samples/README.md` and we should include that in the docs?

**Comment by pgerv12** [comment]
I'll update with what I have and we can discuss where we should go from there.

**Review by ghost — APPROVED** [review]
Looks good to me.

**Pull request merged**